docassemble/MOHUDEvictionProject/data/questions/MOHUDEvictionProject.yml

```diff
@@ -508,6 +599,21 @@ code: |
 code: |
   motion_to_shorten_time_attachment.enabled = wants_discovery and not original_hearing_date_past
 ---
+code: |
+  assembled_documents = list()
+  if "Answer" not in assembled_documents:
+    assembled_documents.append("Answer")
+  if "Instructions" not in assembled_documents:
+    assembled_documents.append("Instructions")
+  if eviction_motion_to_continue_attachment.enabled and "Motion to Continue" not in assembled_documents:
+    assembled_documents.append("Motion to Continue")
+  if eviction_motion_for_leave_attachment.enabled and "Motion for Leave" not in assembled_documents:
+    assembled_documents.append("Motion for Leave")
+  if eviction_discovery_attachment.enabled and "Discovery" not in assembled_documents:
+    assembled_documents.append("Discovery")
+  if motion_to_shorten_time_attachment.enabled and "Motion to Shorten Time" not in assembled_documents:
+    assembled_documents.append("Motion to Shorten Time")
```

Comment:
This might not be fully run if there's an exception mid-way--you could use `assembled_documents_tmp`